src/c/core/session/stream/input_reliable_stream.c

```diff
@@ -95,7 +94,7 @@ bool uxr_next_input_reliable_buffer_available(uxrInputReliableStream* stream, uc
     return available_to_read;
 }
 
-void uxr_buffer_acknack(const uxrInputReliableStream* stream, ucdrBuffer* ub)
+void uxr_buffer_acknack(uint8_t stream_id, const uxrInputReliableStream* stream, ucdrBuffer* ub)
```

Comment:
To be according with the rest of the style, could the `uxrInputReliableStream* stream` be the first argument of the function. Usually, the first paramenter of the function is always the entity on which the action is made (as the `this` parameter in C++)

Reply:
I like your style. Changed!